# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  b86703ee99b724f01183a4d294c6b7c69d963576
Branding 9: Hack appname some more, and even more.

diff --git a/xpcom/glue/XREAppData.cpp b/xpcom/glue/XREAppData.cpp
--- a/xpcom/glue/XREAppData.cpp
+++ b/xpcom/glue/XREAppData.cpp
@@ -51,11 +51,9 @@ void XREAppData::GetDBusAppName(nsACStri
   const char* env = getenv("MOZ_DBUS_APP_NAME");
   if (env) {
     aName.Assign(env);
   } else {
-    aName.Assign(name);
-    ToLowerCase(aName);
-    SanitizeNameForDBus(aName);
+    aName.AssignLiteral("betterbird");
   }
 }
 
 }  // namespace mozilla
